src/@sdk/react/mutations.ts

```diff
@@ -17,6 +17,14 @@ export const useSignIn = (
     update: (proxy, data) => {
       if (data.data.tokenCreate.token) {
         setAuthToken(data.data.tokenCreate.token);
+        if (window.PasswordCredential && variables) {
```

Comment:
If `PasswordCredential` exists in global window, does that mean `navigator.credentials` exists too? If not maybe we should double check:
```
if (navigator.credentials && navigator.credentials.preventSilentAccess) {
    // Credential Management API is supported!
}
```